R/stat-smooth.r

```diff
@@ -127,12 +124,25 @@ StatSmooth <- ggproto("StatSmooth", Stat,
         xseq <- seq(range[1], range[2], length.out = n)
       }
     }
-    # Special case span because it's the most commonly used model argument
+
+    ## Special case span because it's the most commonly used model argument
```

Comment:
It looks like you just changed `#` to `##`. This makes me suspect that you're using ESS?

Reply:
Yes!